fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeFileStats.java

```diff
@@ -105,38 +103,44 @@ public long getRecordCount() {
         return recordCount;
     }
 
-    public void fillColumnStats(ColumnStatistic.Builder builder, Column col) {
-        // Set default value
+    public ColumnStatistic fillColumnStats(Column col) {
+        ColumnStatistic.Builder builder = ColumnStatistic.builder();
+        String colName = col.getName();
+        Type colType = col.getType();
+
+        // set default value
         builder.setNullsFraction(0);
-        builder.setAverageRowSize(col.getType().getTypeSize());
+        builder.setAverageRowSize(colType.getTypeSize());
         builder.setDistinctValuesCount(1);
         builder.setType(ColumnStatistic.StatisticType.UNKNOWN);
 
         if (schema == null) {
-            return;
+            return builder.build();
+        }
+        StructField field = schema.get(colName);
+        if (field == null) {
+            return builder.build();
         }
-
-        String colName = col.getName();
         if (!nonPartitionPrimitiveColumns.contains(colName)) {
-            return;
+            return builder.build();
         }
 
         if (minValues != null) {
             if (col.getType().isStringType()) {
-                String minString = getBoundStatistic(colName, minValues).toString();
+                String minString = minValues.get(colName).toString();
```

Comment:
why not use getBoundStatistic?

Reply:
Currently `getBoundStstatis` can't convert string value